3_javascript/javascript1/week1/future-age/futureAge.js

```diff
@@ -0,0 +1,7 @@
+function calculateAge() {
+  const yearOfBirth = document.getElementById("yearOfBirth").value;
+  const yearFuture = document.getElementById("yearFuture").value;
+  const p = document.getElementById("yourAge");
```

Comment:
I suggest giving the variable a more meaningful and understandable name. As p stands for paragraph, you can just call it a paragraph :smiley:
```suggestion
  const paragraph = document.getElementById("yourAge");
```